examples/Demo/Shared/Pages/Search/Examples/SearchImmediate.razor

```diff
@@ -0,0 +1,110 @@
+﻿@using System.Timers;
+
+<FluentStack Orientation="Orientation.Vertical" VerticalGap="10">
+
+    @* Immediate Delay *@
+    <FluentNumberField @bind-Value="_immediateDelay"
+                       TValue="int"
+                       Label="Immediate Delay"
+                       Placeholder="Delay"
+                       Min="0"
+                       Max="2000"
+                       Step="100" />
+
+    @* Search Box *@
+    <FluentSearch @bind-Value="_searchValue"
+                  @bind-Value:after="OnSearch"
+                  Immediate
```

Comment:
For "clean code", could you use the `Immediate="true"` syntax?

Reply:
Done.